specificatie/openapi.yaml

```diff
@@ -16,33 +16,33 @@ paths:
     get:
       operationId: ingeschrevenNatuurlijkPersonen
       description: |
-        Het ophalen van een collectie ingeschreven personen inclusief het verblijfsadres, ouders, partners, kinderen en reisdocumenten. Het betreft alleen actuele gegevens van de betreffende ingeschreven personen. Ten minste één van de volgende combinaties van parameters moet zijn opgenomen:
+        Het ophalen van een collectie ingeschreven personen inclusief het verblijfsadres, ouders, partners, kinderen en reisdocumenten. Het betreft alleen actuele gegevens van de betreffende ingeschreven personen. Ten minste een van de volgende combinaties van parameters moet zijn opgenomen:
 
 
         1.  Persoon
-            -  geboorte__datum
-            -  naam__geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/features/wildcard.feature) )
+            -  geboortedatum
```

Comment:
Hadden we ook besloten de dubbele underscore uit query parameters te halen? Kan ik me niet herinneren. Is dit consequentie van het doorvoeren van camelCase? Hebben we in BRK niet zo gedaan, daar hebben we de dubbele underscore nog wel

Reply:
Deze Design Decision (DD1.9) is i.i.g. niet aangepast dus volgens mij heb je gelijk.